src/main/scala/cromwell/binding/values/WdlFloatValue.scala

```diff
@@ -0,0 +1,28 @@
+package cromwell.binding.values
+
+import cromwell.binding.types.WdlFloatType
+
+case class WdlFloatValue(value: Float) extends WdlPrimitiveValue {
+  val wdlType = WdlFloatType
+  def add(rhs: WdlValue): WdlValue = {
+    rhs match {
+      case r:WdlFloatValue => WdlFloatValue(value + r.value)
```

Comment:
This is not fully implemented at the moment.  I did bare minimum to get this merged in so Miguel and I don't collide too much.  I'll finish implementing these in another PR